packages/cli/bin/run.js

```diff
@@ -2,6 +2,6 @@
 
 // eslint-disable-next-line unicorn/prefer-top-level-await
 ;(async () => {
-  const oclif = await import('@oclif/core')
+  const oclif = require('@oclif/core')
```

Comment:
@nicolasbrugneaux i know you said this works. just checking that you completely cleaned before rebuilding re packing etc

Reply:
why this change though at all?